src/NetscriptFunctions/Infiltration.ts

```diff
@@ -41,8 +41,8 @@ export function NetscriptInfiltration(player: IPlayer): InternalAPI<IInfiltratio
     };
   };
   return {
-    getPossibleLocations: () => (): string[] => {
-      return getLocationsWithInfiltrations.map((l) => l + "");
+    getPossibleLocations: () => (): any[] => {
+      return getLocationsWithInfiltrations.map((l) => ({ city: l.city, name: l.name }));
```

Comment:
(Long time player; first time commenter :wink:  )

I've been trying out this branch, and this method signature doesn't really help much in the editor (or VSCode when loading `bitburner.d.ts`).

At the very least could this (and other references) be changed to
```suggestion
    getPossibleLocations: () => (): {city: string, name: string}[] => {
      return getLocationsWithInfiltrations.map((l) => ({ city: l.city, name: l.name }));
```

Ideally, it could be `{city: Cityname | null, name: Location}`, but that would require exporting those `enum` values and I'm not sure what this project's stance is on that (it looks like a few stock related ones are exported, but no others)

Reply:
hmm i think i tried this and could not or maybe it was lost in a merge in the past ill try again.